generator/internal/codegen_utils.cc

```diff
@@ -140,6 +140,26 @@ ProcessCommandLineArgs(std::string const& parameters) {
   return command_line_args;
 }
 
+std::string CopyrightLicenseFileHeader() {
+  static auto constexpr kHeader =  // clang-format off
+  "// Copyright 2020 Google LLC\n"
```

Comment:
@scotthart @coryan how about we at least file an issue about making sure the copyright year is updated or something correctly?

Reply:
That SGTM.

Reply:
#4884 created